app/Makefile

```diff
@@ -19,5 +21,10 @@ dev-build-server:
 	cd ./server; \
 	npm run build:dev
 
+test-ui:
+	cd ./ui; \
+	npm test
+
 .PHONY: \
 	run \
+	test \
```

Comment:
> Is there a reason why we're using half make and half npm scripts?

Not any reason particularly. We've partially made the move from `npm` scripts exclusively to `make`. We need to finish that transition :)